packages/modular-scripts/src/cli.ts

```diff
@@ -39,6 +41,7 @@ function execSync(
     stdin: process.stdin,
     stderr: process.stderr,
     stdout: process.stdout,
+    cleanup: true,
```

Comment:
I guess this is the issue we were talking about when it comes to passing through `SIGTERM` etc. ?

Reply:
Yeah, but even this doesn't really fix it. Still good to have so I left it in.